### ISSUE TITLE
Volume does not accept decimal strings, when in game the command does

### ISSUE THREAD

**Issue by TerdyTheTerd** [Ajneb97/Codex#17]
The volume config only allows for integer values, while the in game command allows for decimal strings like 0.1. The plugin needs to be updated to allow for this, so that we can not deafen players when playing audio.

**Comment by Ajneb97** [comment]
You are right. I am thinking on recoding Codex completely, to fix these kind of bugs and implement new features, it will take some time but for now I can't make tiny updates, I prefer to release a new full version when it is ready!
